packages-node/providence-analytics/src/program/services/InputDataService.js

```diff
@@ -205,6 +205,33 @@ function multiGlobSync(patterns, { keepDirs = false, root } = {}) {
   return Array.from(res);
 }
 
+function stripDotSlashFromLocalPath(localPathWithDotSlash) {
+  return localPathWithDotSlash.replace(/^\.\//, '');
+}
+
+function normalizeLocalPathWithDotSlash(localPathWithoutDotSlash) {
+  if (!localPathWithoutDotSlash.startsWith('.')) {
+    return `./${localPathWithoutDotSlash}`;
+  }
+  return localPathWithoutDotSlash;
+}
+
+/**
+ * @param {{val:object|string;nodeResolveMode:string}} opts
+ * @returns {string}
+ */
+function getStringOrObjectValOfExportMapEntry({ val, nodeResolveMode, packageRootPath }) {
+  if (typeof val !== 'object') {
```

Comment:
typeof null === 'object'

Reply:
yeah indeed, we assume people to not have smth like `"exports": { "./": null }` in their export map. We could consider check general validity of exports obj before this method?

Reply:
Will add an input validation later 👍